### PR TITLE
Don't allow equipping while in ring

### PR DESCRIPTION
**Pull request by iowillhoit**

### CODE REVIEW

**Comment by iowillhoit** [comment]
screenshot 2017-12-19 10 24 32

File: monsters/index.js [hunk 1]
```diff
@@ -136,10 +136,16 @@ ${getChoices(genders)}`,
 		.then(() => new Monster(options));
 };
 
-const equip = (deck, monster, cardSelection, channel) => {
+const equip = (deck, monster, cardSelection, channel, ring) => {
 	const cards = [];
 	const { cardSlots } = monster;
 
+	if (ring.monsterIsInRing(monster)) {
```
Comment (tweettypography) [inline monsters/index.js]: You could simplify this a lot and avoid passing the ring around if you did `monster.inEncounter` instead

Comment (iowillhoit) [reply monsters/index.js]: oh nice! i should have just asked :P

**Comment by RodEsp** [comment]
Aw man... 👎

**Comment by iowillhoit** [comment]
Your tricks are over Rodrigo!

**Pull request merged**